src/Microsoft.OData.Edm/Csdl/Semantics/CsdlSemanticsModel.cs

```diff
@@ -303,10 +303,6 @@ public override IEnumerable<IEdmVocabularyAnnotation> FindDeclaredVocabularyAnno
                 {
                     return semanticsReference.InlineVocabularyAnnotations;
                 }
-                else
-                {
-                    Enumerable.Empty<IEdmVocabularyAnnotation>();

```

Comment:
I think this was intended to return an empty enumerable.
So it should probably be `return Enumerable.Empty<IEdmVocabularyAnnotation>();`
What do you think @xuzhg @mikepizzo ?

Reply:
Done